src/functions/ensureEnv.ts

```diff
@@ -0,0 +1,161 @@
+import * as clc from "cli-color";
```

Comment:
Why can't this just be in src/functions/env?

Reply:
Could be. I just preferred to keep the check&enable logic on a separate file. Is there a reason to prefer keeping the logic together?

Reply:
Largely because I'm trying to add some general structure to the project layout so that an engineer ramping up is guided to see and understand the codebase in units. I'd probably request this either be the same `env` file or that we have a separate `env/index.ts` and `env/ensure.ts`.

Reply:
Thanks for clarifying. Will put things together in `env.ts`.